app/uk/gov/hmrc/serviceconfigs/persistence/FrontendRouteRepo.scala

```diff
@@ -65,7 +65,8 @@ class FrontendRouteRepo @Inject()(mongo: ReactiveMongoComponent)
     val s = Json.obj(
       "service" -> Json.toJson[String](frontendRoute.service),
       "environment" -> Json.toJson[String](frontendRoute.environment),
-      "frontendPath" -> Json.toJson[String](frontendRoute.frontendPath)
+      "frontendPath" -> Json.toJson[String](frontendRoute.frontendPath),
+      "routesFile" -> Json.toJson[String](frontendRoute.routesFile)
     )
```

Comment:
Define a Json writes for FrontendRoute?

Reply:
t=The json is used as a partial filter to the update query, not for writing the whole object, but will rename 's' to be clearer